roda-ui/roda-wui/src/main/resources/config/theme/fonts/fonts.css

```
@@ -133,3 +133,40 @@
+  font-display: swap;
+  src: url(../../api/v2/themes?resource-id=fonts/BarlowCondensed/BarlowCondensed-SemiBold.woff2) format('woff2');
+}
+@font-face {
+  font-family: 'Barlow Condensed';
+  font-style: normal;
+  font-weight: 700;
+  font-display: swap;
+  src: url(../../api/v2/themes?resource-id=fonts/BarlowCondensed/BarlowCondensed-Bold.woff2) format('woff2');
+}
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.9.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)